.gitignore

```diff
@@ -4,5 +4,11 @@ __pycache__/
 *.egg-info/
 dist/
 build/
-.direnv/
 .vscode/
+
+# direnv environment files
+.direnv/
+
+# printer output
+prints/**
```

Comment:
This should probably be removed (if my proposal of using `/tmp/receipt_printer` is accepted).